gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -1072,23 +1074,34 @@ message Stop {
   extensions 9000 to 9999;
 }
 
+// NOTE: This field is still experimental, and subject to change. It may be formally adopted in the future.
 message TripModifications {
 	message Modification {
-    // The first stop sequence of the original trip that is to be affected by this modification.
-		optional uint32 start_stop_sequence = 1;
-
-    // The number of stops which are canceled and replaced by the replacement_stops. May be zero to indicate no skipped stops.
-		optional uint32 num_stops_replaced = 2;
-
-    // The number of seconds of delay to add to all departure and arrival times following the end of this modification. If multiple modifications apply to the same trip, the delays accumulate as the trip advances. 
+    // The stop selector of the first stop of the original trip that is to be affected by this modification. 
+    // If `end_stop_selector` is not provided, no stop will be removed and the stop from the `start_stop_selector` 
+    // is only provided as a reference stop for `travel_time_to_stop` calculations.
```

Comment:
I would maybe omit these details about the end stop selector here, since you cover what happens if the end_stop_selector is not provided below. 

I think the main purpose of having it here is to help clarify what stop should be included as the "first affected" in particular when no stops are skipped. Consider instead addressing this specifically with something like "if no stops are skipped, this should refer to the stop immediately after the modification" or something like that.

Reply:
Indeed, I modified the description!